packages/firestore/CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
-# Unreleased (1.8.0)
+# Unreleased
+- [feature] Added support for strongly typed collections, documents, and
```

Comment:
I don't think average Joe Schmidt (a JavaScript user) will be able to decipher the true impact of this change. You should point out that users can now store and retrieve custom classes in Firestore documents.